packages/databricks-vscode/resources/python/notebook.workflow-wrapper.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Could we get rid of this file and create the JSON dynamically? the cell source should be identical to the workflow wrapper and it's easy to forget to keep them in sync.,